### PR TITLE
feat(raster): add custom colormap for EPA dataset

### PR DESCRIPTION
**Pull request by vincentsarago**
ref https://github.com/NASA-IMPACT/veda-config-ghg/issues/203

cc @j08lue

### CODE REVIEW

File: raster_api/runtime/setup.py [hunk 1]
```diff
@@ -8,7 +8,9 @@
 inst_reqs = [
     "boto3",
     "titiler.pgstac==0.8.0",
-    "titiler.extensions[cogeo]>=0.15.0,<0.16",
+    "titiler.core>=0.15.1,<0.16",
```
Comment (vincentsarago) [inline raster_api/runtime/setup.py:11]: `titiler.core.dependencies.create_colormap_dependency` was added in `0.15.1`

**Comment by j08lue** [comment]
Awesome, thanks @vincentsarago. 

We need to test this somehow and verify with the data provider that the color map is correct.

We should also rename the colormap to `epa-ghgi-ch4`, please.

I'll try to organize for someone to complete these steps.

File: raster_api/runtime/src/cmap_data/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+alphas_slope = np.abs(np.linspace(0, 1.0, slopen))
+alphas_stable = np.ones(3003-slopen)
+alphas = np.concatenate((alphas_slope, alphas_stable))
+my_cmap._lut[:,-1] = alphas
+my_cmap.set_under('white', alpha=0)
+
+x = np.linspace(0, 1, 256)
+cmap_vals = my_cmap(x)[:, :]
+cmap_uint8 = (cmap_vals * 255).astype('uint8')
+np.save("epa.npy", cmap_uint8)
```
Comment (vincentsarago) [inline raster_api/runtime/src/cmap_data/README.md:29]:
```suggestion
np.save("epa-ghgi-ch4.npy", cmap_uint8)
```

**Comment by anayeaye** [comment]
The latest is running in the test stack here: https://test-raster.delta-backend.com/docs

File: raster_api/runtime/setup.py (hunk 1 above)
Comment (anayeaye) [reply raster_api/runtime/setup.py:11]: @vincentsarago I noticed the bidx/asset_bidx change in the titiler changelog. Will this [`asset_bidx` parameter usage](https://github.com/NASA-IMPACT/veda-config/blob/main/datasets/frp-max-thomasfire.data.mdx#L25) still be supported in v0.15.4+?
```
    sourceParams:
      asset_bidx: cog_default|1
```

Comment (vincentsarago) [reply raster_api/runtime/setup.py:11]: `asset_bidx` is the good way to go 👍 

it enable more granular band selection. if you endup doing `asset_bidx: [asset1|1, asset2|1]` , using `bidx=1` will just be easier to write.

Comment (anayeaye) [reply raster_api/runtime/setup.py:11]: Never mind, I just confirmed the current usage is not impacted!

**Review by anayeaye — APPROVED** [review]

File: raster_api/runtime/src/cmap_data/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+alphas_slope = np.abs(np.linspace(0, 1.0, slopen))
+alphas_stable = np.ones(3003-slopen)
+alphas = np.concatenate((alphas_slope, alphas_stable))
+my_cmap._lut[:,-1] = alphas
+my_cmap.set_under('white', alpha=0)
+
+x = np.linspace(0, 1, 256)
+cmap_vals = my_cmap(x)[:, :]
+cmap_uint8 = (cmap_vals * 255).astype('uint8')
+np.save("epa-ghgi-ch4.npy", cmap_uint8)
```
Comment (anayeaye) [inline raster_api/runtime/src/cmap_data/README.md:29]: should we have a `epa-ghgi-ch4.npy` in cmap_data instead of `epa.npy` checked for this branch?

Comment (vincentsarago) [reply raster_api/runtime/src/cmap_data/README.md:29]: @anayeaye the colormap name in the application should be `epa-ghgi-ch4`

Comment (vincentsarago) [reply raster_api/runtime/src/cmap_data/README.md:29]: 🤦 oh I forgot to rename the file itself 🤦 🤦 🤦 🤦 🤦 🤦

**Pull request merged**